configs/compute/1gpu.cfg

```diff
@@ -1,13 +1,13 @@
-numGameThreads = 32
+numGameThreads = 128
 
 # GPU Settings-------------------------------------------------------------------------------
 # 1xA6000
 # 2 sockets of AMD EPYC 7763 64-Core Processors = 128 physical cores
 #                                                 each core has 2 threads
 
-nnMaxBatchSize = 32
-nnCacheSizePowerOfTwo = 24
-nnMutexPoolSizePowerOfTwo = 18
+nnMaxBatchSize = 128
+nnCacheSizePowerOfTwo = 21
```

Comment:
Partly this was because we also use this config for `match` where there may be many nets, in which case we don't want to have lots 12GB caches. 
I don't think the cache size gives a huge performance improvement but I'll run a benchmark and report back tomorrow.

Reply:
Doesn't seem to give much of a performance bonus if any: https://www.notion.so/chaiberkeley/debugging-victimplay-throughput-being-lower-than-expected-bc376db69a5a4343bd6c609dcec1ca4b?pvs=4#468b03eb47d8495e976d810117cfa7f1

Perf difference between nnCacheSizePowerOfTwo = 21 vs. 22 was 3% (but not statistically significant), and between 21 vs. 23 was 0.6%